_posts/elements/2015-07-10-layouts.md

```diff
@@ -0,0 +1,322 @@
+
+info: Grid-based preset layouts that can be used for larger alignment for bands of a view's UI or specifically for the view at large.
+
+example: layouts.html
+---
+
+<h3 class="hd-6 example-set-hd">Full Layout</h3>
+<div class="example-set example-layout">
+    <div class="layout-full layout">
+
+        <main class="layout-col layout-col-a">
+            <span class="example-label">col-a</span>
+        </main>
+
+        <aside class="layout-col layout-col-b">
+            <span class="example-label">col-b</span>
+        </aside>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">1/4 + 3/4 Layout</h3>
+<div class="example-set example-layout">
+    <div class="layout-1q3q layout">
+
+        <main class="layout-col layout-col-a">
+            <span class="example-label">col-a</span>
+        </main>
+
+        <aside class="layout-col layout-col-b">
+            <span class="example-label">col-b</span>
+        </aside>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">1/4 + 3/4 Layout (Flush)</h3>
+<div class="example-set example-layout">
+    <div class="layout-1q3q layout-flush">
+
+        <main class="layout-col layout-col-a">
+            <span class="example-label">col-a</span>
+        </main>
+
+        <aside class="layout-col layout-col-b">
+            <span class="example-label">col-b</span>
+        </aside>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">1/4 + 3/4 Layout (Reversed)</h3>
+<div class="example-set example-layout">
+    <div class="layout-1q3q layout-reversed">
+
+        <main class="layout-col layout-col-a">
+            <span class="example-label">col-a</span>
+        </main>
+
+        <aside class="layout-col layout-col-b">
+            <span class="example-label">col-b</span>
+        </aside>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">1/4 + 1/2 + 1/4 Layout</h3>
+<div class="example-set example-layout">
+    <div class="layout-1q1h1q layout">
+
+        <nav class="layout-col layout-col-a">
+            <span class="example-label">col-a</span>
+        </nav>
+
+        <main class="layout-col layout-col-b">
+            <span class="example-label">col-b</span>
+        </main>
+
+        <aside class="layout-col layout-col-c">
+            <span class="example-label">col-c</span>
+        </aside>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">1/4 + 1/2 + 1/4 Layout (Flush)</h3>
+<div class="example-set example-layout">
+    <div class="layout-1q1h1q layout-flush">
+
+        <nav class="layout-col layout-col-a">
+            <span class="example-label">col-a</span>
+        </nav>
+
+        <main class="layout-col layout-col-b">
+            <span class="example-label">col-b</span>
+        </main>
+
+        <aside class="layout-col layout-col-c">
+            <span class="example-label">col-c</span>
+        </aside>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">1/4 + 1/2 + 1/4 Layout (Reversed)</h3>
```

Comment:
Aha! I fooled you on this. There is one difference, this layout has a class of ``.layout-reversed`` which reverses the flow of the layout (without needing to shift any DOM order at all - that's why no other markup has changed). 

I thought this way of reversing a layout was a nice option especially if its just for presentation. Seem beneficial to you or should we nix reverse layouts and just have folks adjust the DOM?